dist/README.UPDATERS

```diff
@@ -1,3 +1,54 @@
+For Updaters to OBS 2.9 from OBS 3.0
```

Comment:
This should be "to OBS 3.0 from OBS 2.9" or "from OBS 2.9 to OBS 3.0"

Reply:
oh, right:D

Reply:
@eduardoj fixed